pkg/clusterid/creator.go

```diff
@@ -26,19 +28,32 @@ func (c *clusterIDCreator) NeedLeaderElection() bool {
 }
 
 func (c *clusterIDCreator) create() error {
-	ctx := user.Ctx(context.Background(), user.ControlPlane)
+	contexts, err := iterator.CustomIterator()
+	if err != nil {
+		log.Error(err, "could not get contexts")
+	}
+	var allErrors error
+	for _, ctx := range contexts {
+		err := Create(user.Ctx(ctx, user.ControlPlane), c.configManager)
+		if err != nil {
+			allErrors = multierror.Append(allErrors, err)
+		}
+	}
+	return allErrors
+}
```

Comment:
Is there more than one clusterId in the multi tenant use case?

Reply:
There is only one global cp, let's revert this from tenant-awareness